curbs/examples.md

```diff
@@ -26,31 +26,46 @@ A [Query Curb Zones](/curbs#query-curb-zones) example of `/curbs/zones` with min
   "currency": "USD",
   "author": "City of Metropolis",
   "license_url": "https://creativecommons.org/licenses/by/4.0/",
-  "data": [
-    {
-      "zones": [
-        {
-          "curb_zone_id": "7d8a5885-e949-4ac9-afb7-fa4d43b68530",
-          "geometry": {
-            "type": "Polygon",
-            "coordinates": [[
-              [-73.982105, 40.767932],
-              [-73.973694, 40.764551],
-              [-73.949318, 40.796918],
-              [-73.958416, 40.800686],
-              [-73.982105, 40.767932]
-            ]]
-          },
-          "curb_policy_ids": [
-            "cd0996d7-3765-4f0b-a72e-7caf7cf3fe21"
-          ],
-          "published_date": 1552678594428,
-          "last_updated_date": 1552678594428,
-          "start_date": 1552678594428
-        }
-      ] 
-    }
-  ]
+  "data": {
+    "zones": [
+      {
+        "curb_zone_id": "7d8a5885-e949-4ac9-afb7-fa4d43b68530",
+        "geometry": {
+          "type": "Polygon",
+          "coordinates": [
```

Comment:
Can you format this down to this just to be more compact on the page? 

            "coordinates": [[
              [-73.982105, 40.767932],
              [-73.973694, 40.764551],
              [-73.949318, 40.796918],
              [-73.958416, 40.800686],
              [-73.982105, 40.767932]
            ]]

Reply:
Done in [a2a694a](https://github.com/openmobilityfoundation/curb-data-specification/pull/132/commits/a2a694ab9210a7dae1fc325bb7739a975c92e38c)